addons/hr_expense/models/hr_expense.py

```diff
@@ -435,6 +436,30 @@ def _compute_employee_id(self):
             for expense in self:
                 expense.employee_id = self.env.user.with_company(expense.company_id).employee_id
 
+    @api.depends('attachment_ids')
+    def _compute_same_receipt_expense_ids(self):
+        for expense in self:
+            expense.same_receipt_expense_ids = [Command.clear()]
```

Comment:
```suggestion
       self.same_receipt_expense_ids = [Command.clear()]
```
? Should work the same :thinking: 
Not sure about how to improve this method but pretty sure it could :thinking: